tests/functional/response_func_test.py

```diff
@@ -43,7 +43,7 @@ def test_primitive_types_returned_in_response(httprettified, swagger_dict):
         'number': 3.4,
         'boolean': True
     }
-    for rtype, rvalue in rtypes.items():
+    for rtype, rvalue in list(rtypes.items()):
```

Comment:
Same as above comment, you can iterate rtypes.items().

Reply:
done